clang/test/SemaSYCL/intel-max-work-group-size.cpp

```diff
@@ -96,8 +96,21 @@ void f9() {}
 [[intel::max_work_group_size(16, 16, 16)]] void f14(); // expected-note {{conflicting attribute is here}}
 [[sycl::reqd_work_group_size(64, 64, 64)]] void f14(); // expected-error{{'reqd_work_group_size' attribute conflicts with 'max_work_group_size' attribute}}
 
-[[intel::max_work_group_size(2, 2, 2)]] void f15();
-[[sycl::reqd_work_group_size(2, 2, 2)]] void f15(); // OK
+[[cl::reqd_work_group_size(1, 2, 3)]] // expected-warning {{attribute 'cl::reqd_work_group_size' is deprecated}} \
```

Comment:
I'm unclear what the behavior here should be. We are using cl spelling in SYCL mode.

Reply:
As per discussion with @AaronBallman in the thread:https://github.com/intel/llvm/pull/3728#discussion_r632653354
 ``I read that suggestion as meaning: __attribute__((reqd_work_group_size)) is only available in OpenCL mode and follows the OpenCL rules. [[sycl::reqd_work_group_size]], [[cl::reqd_work_group_size]], and [[intel::reqd_work_group_size]] are only available in SYCL modes and follow the SYCL rules.``

So [[cl::reqd_work_group_size]] is following SYCL rules here.